tss-esapi-sys/build.rs

```diff
@@ -149,6 +274,21 @@ pub mod tpm2_tss {
                             .clang_arg(tss2_tcti_tbs.include_dir_arg())
                             .header(tss2_tcti_tbs.header_file_arg());
                     }
+                    #[cfg(windows)]
+                    {
+                        let mut clang_args: Vec<String> = Vec::new();
```

Comment:
move this into a separate function as well. e.g.
`fn platform_args() -> Option<Vec<&str>>`  and for all other platforms we currently just return None there.

and then do another 
```rust 
if let Some(platform_args) = platform_args() {
   builder = builder.clang_args(platform_args)
}
```